source/FreeRTOS_Sockets.c

```diff
@@ -4780,6 +4780,55 @@ BaseType_t xSocketValid( const ConstSocket_t xSocket )
 #endif /* ipconfigUSE_TCP */
 /*-----------------------------------------------------------*/
 
+#if ( ipconfigUSE_SetSocketID != 0 )
+
+/**
+ * @brief Set the value of the SocketID of a socket.
+ * @param[in] xSocket: The socket whose ID should be set.
+ * @param[in] pvSocketID: The new value for the SocketID.
+ * @return Zero if the socket was valid, otherwise -EINVAL.
+ */
+    BaseType_t xSocketSetSocketID( const Socket_t xSocket,
```

Comment:
Seems like this is an API. Why not stick to the convention of starting the name with `FreeRTOS_`?
Something like `FreeRTOS_SetSocketID`?

Reply:
A disadvantage of `FreeRTOS_SetSocketID` would be that the name doesn't carry the type prefixes, like "x" and "pv".